source/codegen/templates/service_helpers.mako

```diff
@@ -208,6 +208,14 @@ ${initialize_standard_input_param(function_name, parameter)}\
         ${parameter_name}_request.end(),
         std::back_inserter(${parameter_name}),
         [](auto x) { return x ? VI_TRUE : VI_FALSE; });
+% elif c_type == 'ViInt16[]':
+      auto ${parameter_name}_request = ${request_snippet};
+      std::vector<${c_type_underlying_type}> ${parameter_name};
+      std::transform(
+        ${parameter_name}_request.begin(),
+        ${parameter_name}_request.end(),
+        std::back_inserter(${parameter_name}),
+        [](auto x) { return (${c_type_underlying_type})x; });
```

Comment:
If you add a backslash at the end of this line, you wont see the extra line in the cpp file.

Reply:
Done.